### DISCUSSION TITLE
4.4 kWp / 5.12 kWh System Share

### DISCUSSION THREAD

**Discussion by ahiggs1** [hoylabs/OpenDTU-OnBattery#1460] [Show and tell]
Hi,

[I'd like to share a PV setup I completed with 4.4kWp controlled by the OpenDTU-OnBattery software](https://docs.google.com/drawings/d/1BY93ZtKmbnio8FbEMruhaYkgwnzNxRTUlkffQhfyHl4/edit?usp=sharing). I started with a GreenSolar Gen3 2.2kWh battery and 4x 450W panels, but realized that their Marstek smart meter is too imprecise to actually achieve a zero-feed in. Furthermore, while the battery states up to 800W input per PV string, the internal MPPT stops at around 13 A. With two 450W panels (LNVK-450M: 35.0 Vmp, 12.7 A) in parallel, the highest output I saw was ~500W. The true 800W can only be achieved if the input voltage is exactly 59 V, making this configuration very inefficient...

I decided to redo the system with a 5.12kWh Pytes V5a battery and added an extra six 440W panels to comply with the requirements for 0% MWST in Austria. All 10 panels are oriented south at 17° inclination on a flat roof. The roof itself is oriented southwest, making this not the most efficient use of space. Two Victron 150/45 MPPTs connect to the Pytes through a Lynx distributor. To avoid cable salad and the [cable length mismatch issue](https://github.com/hoylabs/OpenDTU-OnBattery/wiki/Kabell%C3%A4nge%28n%29-zwischen-Batterie-und-Wechselrichter), I disassembled an Hoymiles HM-800 and internally connected the two MPPT inputs. At full load, there is a <0.1A mismatch in string currents. The MPPTs, battery, and inverter are controlled by the OpenDTU fusion PCB using grid consumption values from a Shelly EM3Pro.

The total cost was around 4000€, however nearly 1000€ of this came from installation material and the panel holders. Drilling into the roof was not an option and I did not want to use the questionable flimsy L-brackets that typically come with flat-roof mounting kits, so I purchased the [ValkBox3](https://www.valksolarsystems.com/de/pv-montagesysteme/kits/flachdacher/standard/valkbox3) which includes certification for wind loading. 

In November, it produced 188kWh, with maximum daily yield around 10kWh. At this rate of 1.5€/day (0.25€/kWh), the estimated payback time is a little over 7 years. 

If anyone has questions or is considering setting up a similar system, please feel free to reach out!

Thanks for reading,
Ansel

**Comment by schlimmchen** [comment]
> I disassembled an Hoymiles HM-800 and internally connected the two MPPT inputs. At full load, there is a <0.1A mismatch in string currents.

Interesting approach! However, I am glad I did not try this on my father's HMS-2000, which died and I had to return it under warranty :grin:

> with maximum daily yield around 10kWh.

Wait, what? That was a particularly sunny day, wasn't it? :wink:

**Comment by ahiggs1** [comment]
The most difficult part was opening the casing, as the glue (?) remains sticky even once it has been sliced through. The desoldering the connection of the original cables to the PCB was actually very manageable even with a cheap Amazon iron. I'm curious how long the HM-800 lasts, it typically runs at 62°C under full 800W load. I tried to mount it to allow air flow up the back panel as well, but it seems like most heat is dissipated on the black (plastic ?) side with the LED, as the PCB has a much better thermal connection to this side. I would be interested in upgrading to a 1600 or a 2250, but I'm not sure how easy it is to get the system approved.

It needs to be sunny in the morning to reach 10kWh, but it produced over 8kWh a total of ~10 times during November. Our heat pump runs only during the day, so the grid consumption is almost always above 800W, allowing the inverter to constantly run at its limit

**Comment by Manos1966** [comment]
> Drilling into the roof was not an option and I did not want to use the questionable flimsy L-brackets that typically come with flat-roof mounting kits, so I purchased the [ValkBox3](https://www.valksolarsystems.com/de/pv-montagesysteme/kits/flachdacher/standard/valkbox3) **which includes certification for wind loading.**

That is a very valid point (no holes on the roof).

What Certification for wind loading do they have? (all this is new to me).

**Reply by ahiggs1** [reply]
Wind loading is EN 1991-1-4, everything is listed here https://www.valksolarsystems.com/media/3408/en-datasheet-certification-solar-mounting-systems-2020.pdf
